devito/ops/transformer.py

```diff
@@ -1,3 +1,29 @@
+from devito.ir.iet import Callable, Expression, FindNodes
+from devito.ops.node_factory import OPSNodeFactory
+
+
+def opsit(trees, count):
+    node_factory = OPSNodeFactory()
+    expressions = []
+    for tree in trees:
+        expressions.extend([
+            Expression(make_ops_ast(i.expr, node_factory))
+            for i in FindNodes(Expression).visit(tree.inner)
+        ])
+
+    arguments = []
+    for exp in expressions:
+        func = [f for f in exp.functions if f.name != "OPS_ACC_size"]
+        for f in func:
+            f.is_OPS = True
+            f.is_LocalObject = True
+        arguments.extend(func)
+
+    callable_kernel = Callable("Kernel{}".format(count), expressions, "void", arguments)
```

Comment:
At this point i think you could use a lambda expression for the first parameter 
(look: https://github.com/opesci/devito/blob/master/devito/yask/utils.py#L69 )

Something like this:
`Callable(namespace['ops-kernel'](count) , expressions, "void", arguments)`